README.md

```diff
@@ -12,18 +12,59 @@ Copy `config.js.template` to `config.js`, and then fill in the variables. Finall
 
 You can enable a simple API Key verification in the `config.js` file. In this case, your requests must include the header `X-API-KEY` with the correct key.
 
-If you are using cURL to test, you can include the header using the `-H` parameter, e.g., `curl -H "X-API-Key: YourKey" http://localhost:8000/0/balance`.
+If you are using cURL to test, you can include the header using the `-H` parameter, e.g., `curl -H "X-API-Key: YourKey" http://localhost:8000/wallet/balance`.
 
 It follows the [Swagger Specification for API Keys](https://swagger.io/docs/specification/authentication/api-keys/).
 
 ## How to use?
 
 Check out the full documentation in the OpenAPI Documentation in `api-docs.js`.
 
+### Start a wallet
+
+Create and start a new wallet.
+
+Parameters:
+
+`key`: key reference of this wallet in the object of created wallets. Used to say which wallet each request is directed to.
+`passphrase`: Optional parameter to start the wallet with a passphrase. If not sent we use empty string.
+`seedKey`: Optional parameter to define which seed (from the object seeds in the config file) will be used to generate the wallet. If not sent we use 'default'.
+
+```bash
+$ curl -X POST --data "key=key"  --data "passphrase=123" --data "seedKey=default" http://localhost:8000/start
+{"success":true}
+```
+
+**All requests below must have a parameter 'key' in the body which wallet must be used.**
```

Comment:
- About having an easy way to use a single wallet:
  Yes, I was thinking about opening a new PR to handle this after we finish this one. I didn't want to make this extensive, since we need it merged by tomorrow morning.
  We could discuss this solution but I think we should have and explicit parameter in the config file to set "multiple": true/false, I think. Then, if it's a single wallet, we already get from default and start it with no need to call `/start`. But we can discuss later, I feel we should do it in a separate PR.

- About the 'key' parameter:
  I don't like having it in the header, feels like wrong to me having a parameter of the request in the header. And the decision of having it as a parameter was exactly to make it easier to handle single/multiple wallets in the same endpoint with no need to create one for each (as you described).

  If you don't like having the GET parameter in the body, I can add it to the query params, since is more common to have. It would be `http://localhost:8000/wallet/status/?key=key`. The reason I decided to use the body was just to handle the parameter in a GET and POST requests the same way. But if you feel having a body parameter for GET is not good, I can do a query param for GET and a body param for POST and handle them separately.

Reply:
I agree with @obiyankenobi. We could get the key from the query string.

```
curl -X GET http://localhost:8000/wallet/status/?key={key}
```

Reply:
I'm not sure it would be a problem having it on the header. I've seem http headers used a lot of times for authentication/authorization. But to be honest, I'm not super familiar with what type of information should be passed on the headers.

As this `key` seems a bit like authentication to me, I'd say it's not a problem to use it in the headers. But I'd wait for more opinions. @jansegre @msbrogli ?

If we go this way, having multiple/single wallets is trivial, as you can still use the store and, if the user does not send a key in the headers, use some default key like `_DEFAULT_`.

Reply:
I kind of like @obiyankenobi's idea of using a header. The `key` is more of a context indicator, and if it is *always* required it may be practical to use a header because several it is common for http libraries to allow adding headers to the "session" or "connection pool" object to be used on every request.

I just don't know if using (reusing) an authentication header is a good idea. Maybe something like `X-Hathor-Key`, `X-Hathor-Wallet`, `X-Wallet-Id` or `X-Hathor-Wallet-Id`. Being picky I think `key` is too generic and may be confused with `key/value`. Custom headers must start with `X-` (per the http spec if I'm not mistaken), and it may be good to start with `X-Hathor` but I don't know if we should go that far.

Reply:
I agree about just using `key`. `X-Wallet-Id` or `X-Hathor-Wallet-Id` both work well.

Reply:
I don't think accessing it from the browser is particularly useful, assuming the only return format is JSON. If it's for development I'm pretty sure it is possible to at least resend requests with edited headers from the development tools.

But I also don't think it's a bad idea to look for multiple places, but for the sake of simpler start I'd prefer to begin with only one way to do it and add other ways in the future (specially if there's feedback that changes our view on this).

Reply:
I agree about looking for multiple places.

My opinion is to keep only in the request for now (in case of a GET I will change to the query and for POST I will keep in the body), since it's already done and after we can add support for the header also.

What do you think?

Reply:
I agree with Jan, I don't see why it's useful to access it in the browser at the moment. I think we should design for the use case we have right now and we can extend it in the future if there are other requirements. So the header solution is still the best, in my opinion.

Reply:
I just think that if we will add support to look for the wallet-id in multiple places, there is no reason to change what it's already coded.

Besides that, I still don't think having it in the header is better than having in a request parameter. I feel it's pretty much the same solution and, since we are thinking about the real use case, people are usually more used to send parameters than creating headers, thinking about a common use. I just don't see why it's better to have it on the header, rather than a request parameter.

Reply:
I agree, we should do it only one way right now. I said we should extend only if it's required in the future.

The advantage I see in using it in the header is that it'll be handled the same way in all requests, either GET or POST. I think the more "correct" way would be having it on the path itself (`/wallet/{id}/start`, `/wallet/{id}/simple-send-tx`, etc), but that would make a bit more complex for handling both multiple and single wallets. That's why I prefer the header solution.

As for keeping this way as it's already coded, while I'm usually ok with that, this will be used externally, so it'll be harder to change moving forward. That's why I think we should get it right before releasing, given that it also wouldn't be much work.

Reply:
I agree that it's harder to change but one of the ideas is to support multiple ways. That's why I think there is no problem if we release like that, it will continue being supported in the future.

Reply:
Added as a header. `X-Wallet-Id`